src/zenml/integrations/evidently/steps/evidently_profile.py

```diff
@@ -127,6 +127,7 @@ def entrypoint(  # type: ignore[override]
         reference_dataset: pd.DataFrame,
         comparison_dataset: pd.DataFrame,
         config: EvidentlyProfileConfig,
+        ignored_columns=Union[None, List[str]],
```

Comment:
This might better be expressed as `Optional[List[str]]`. I think they're equivalent?

Reply:
> This might better be expressed as `Optional[List[str]]`. I think they're equivalent?

Yes you are right, I updated it.